app/assets/javascripts/discourse/app/components/choose-topic.hbs

```diff
@@ -36,7 +36,7 @@
             <span class="topic-categories">
               {{bound-category-link
                 t.category
-                recursive=true
+                ancestors=t.category.predecessors
```

Comment:
I'm not seeing any references to `predecessors` anywhere in our code base... am I missing something?

Reply:
I added predecessors in this change. I would have called it ancestors, but ancestors is already defined and not quite what I want.